### PR TITLE
Add section on finding errors from polkascan and subscan

### LINKED ISSUE #1415: [STALE] maintain-errors.md
Last edit happened 91 days ago 😬

### PR DESCRIPTION
**Pull request by lsaether**
closes #1415

### CODE REVIEW

File: docs/maintain-errors.md [hunk 1]
```diff
@@ -38,6 +38,27 @@ is a live example of the above.
 If you cannot look up the error this way, or there is no message in the `details` field, consult the
 table below.
 
+## Polkascan and Subscan
+
+Polkascan and Subscan show the `ExtrnsicFailed` event when a transaction does not succeed
```
Comment (laboon) [inline docs/maintain-errors.md]:
```suggestion
Polkascan and Subscan show the `ExtrinsicFailed` event when a transaction does not succeed
```

File: docs/maintain-errors.md [hunk 2]
```diff
@@ -38,6 +38,27 @@ is a live example of the above.
...
+([example](https://polkascan.io/polkadot/event/2836233-3)). This event gives us the `error` and
+`index` indices of the error but does not give us a nice message to understand what it means. We
+will look up the error in the codebase ourselves to understand what went wrong.
+
+First, we should understand that the `index` number is the index of the pallet in the runtime from
+which the error originated. The `error` is likewise the index of that pallet's errors which is the
+exact one we're looking for. Both of these indices start counting from 0.
+
+For example, if we have `index` is 5 and `index` is 3 like in the example linked above, we need to
```
Comment (laboon) [inline docs/maintain-errors.md]:
```suggestion
For example, if `index` is 5 and `error` is 3, as in the example linked above, we need to
```

File: docs/maintain-errors.md [hunk 3]
```diff
@@ -38,6 +38,27 @@ is a live example of the above.
...
+look at the runtime for the 4th error in the pallet at index 5.
```
Comment (laboon) [inline docs/maintain-errors.md]:
```suggestion
look at the runtime for the fourth error (index 3) in the sixth pallet (index 5).
```

File: docs/maintain-errors.md [hunk 4]
```diff
@@ -38,6 +38,27 @@ is a live example of the above.
...
+
+By looking at the
+[runtime code](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L914)
+we see that the pallet at index 3 is "Timestamp". Now we will check the timestamp pallet's code
```
Comment (laboon) [inline docs/maintain-errors.md]:
```suggestion
we see that the pallet at index 5 is "Balances". Now we will check the Balances pallet's code
```

File: docs/maintain-errors.md [hunk 5]
```diff
@@ -38,6 +38,27 @@ is a live example of the above.
...
+which is hosted in the Substrate repository. According to
```
Comment (laboon) [inline docs/maintain-errors.md]:
```suggestion
which is hosted in the Substrate repository, and look for the fourth error in the Error enum. According to
```

**Review by laboon — CHANGES_REQUESTED** [review]
Looks like you looked up the error for Balances but then references Timestamps (maybe swapped `error` and `index`).  Also added some clarifications.

**Comment by lsaether** [comment]
Thanks!

**Review by laboon — APPROVED** [review]

**Pull request merged**